src/Background.cpp

```diff
@@ -340,7 +340,7 @@ namespace Background {
 						}
 
 						// Don't process any more sectors if we've generated our quota of stars.
-						if (stars.pos.size() >= starsLimit)
+						if (stars.pos.size() >= (size_t) starsLimit)
```

Comment:
I had a brief look at this a while back; is there actually a good reason that `starsLimit` is not a `size_t`? 

In general, casting indicates "code smell".